<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table</class>
 <widget class="QDialog" name="table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de Estoque</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>691</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cadastro de Produtos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>911</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <column>
    <property name="text">
     <string>CÓDIGO</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOME</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ESTOQUE</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATA DE CADASTRO</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CATEGORIA</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MARCA</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PREÇO DE VENDA</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
